grayskull/__main__.py

```diff
@@ -111,11 +111,18 @@ def main(args=None):
 
     for pkg_name in args.pypi_packages:
         logging.debug(f"Starting grayskull for pkg: {pkg_name}")
-        print_msg(
-            f"{Fore.GREEN}\n\n"
-            f"#### Initializing recipe for "
-            f"{Fore.BLUE}{pkg_name} (pypi) {Fore.GREEN}####\n"
-        )
+        if pkg_name.startswith(("http://", "https://")):
```

Comment:
We can avoid some duplication here by parameterizing the `(pypi)` suffix.

Something like this:

```python
pypi_label = "" if pkg_name.startswith(("http://", "https://")) else " (pypi)"  # note the space!
print_msg(
    f"{Fore.GREEN}\n\n"
    f"#### Initializing recipe for "
    f"{Fore.BLUE}{pkg_name}{pypi_label} {Fore.GREEN}####\n"
)


```